.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Validate & Release
+on:
+  push:
+    tags:
+      - v*
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm install
+      - run: npm run lint
+      - run: npm test
```

Comment:
Because we run the tests in this action it also will need a postgres service to connect to.